src/BenchmarksDriver/Program.cs

```diff
@@ -85,7 +85,7 @@ public static int Main(string[] args)
 
             // ServerJob Options
             var databaseOption = app.Option("--database",
-                "The type of database to run the benchmarks with (PostgreSql, SqlServer or MySql). Default is PostgreSql.", CommandOptionType.SingleValue);
+                "The type of database to run the benchmarks with (PostgreSql, SqlServer or MySql). Default is None.", CommandOptionType.SingleValue);
```

Comment:
Can I ask you to also update the README file with the same change? Thanks.

Reply:
done